### PR TITLE
*: Go 1.7 no longer supported and updated build image to 1.9

### PR DESCRIPTION
**Pull request by ericchiang**
Updates https://github.com/coreos/dex/issues/1118

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,4 @@
-FROM golang:1.8.3-alpine
-
-MAINTAINER Ed Rooth <ed.rooth@coreos.com>
-MAINTAINER Lucas Servén <lucas.serven@coreos.com>
-MAINTAINER Rithu John <rithu.john@coreos.com>
+FROM golang:1.9.2-alpine
```
Comment (rithujohn191) [inline Dockerfile]: why remove the maintainers list?

Comment (ericchiang) [reply Dockerfile]: Sorry should have called that out.

I'd like to clean up the top level MAINTAINER list soon. I don't really see the point of having this list here as well. Our container's don't need our emails in them.

Either way, I can revert this change if we'd like to keep these.

Comment (rithujohn191) [reply Dockerfile]: Its fine to take them out here as long as we maintain the top level one and update the list there.

Comment (ericchiang) [reply Dockerfile]: I'll revert the change from this PR for now

**Pull request merged**